### DISCUSSION TITLE
How can i use this library without this bug?

### DISCUSSION THREAD

**Discussion by arman-bolatuly** [CodePapi/n-krypta#16] [General]
I have this code. I find here a bug. 
` const secretKey = "plt_key";
  const encryptData = encrypt(newPassword, secretKey);
  localStorage.setItem("plt", encryptData);
  const decryptData = decrypt(encryptData, secretKey);

  console.log("encryptData: ", encryptData);

  const newPasswordData = localStorage.getItem("plt");
  let decryptNewPasswordData = [];
  if (newPasswordData) {
    decryptNewPasswordData = decrypt(newPasswordData, secretKey);
  }
  console.log("decryptNewPasswordData: ", decryptNewPasswordData);
`
There giving me these data with bug: 
_decryptNewPasswordData:  [{"password":"h4bUxF'.tc1","success":true,"fullName":"Tom Tomi Tom","username":"tom@mail.ru"},{"password":"Hsso+*67dIS","success":true,"fullName":"Misses Robot MSROBOT","username":"ms_robot@mail.ru"},{"password":"c&PS37o$sDw","success":true,"fullName":"Jane Doe Tom","username":"janen}doe@lbil-rv!~,z"qasswnsd"9"52DjMoO'8*1!-"stcddts!:urue,!gulkNblf"9"Eudar`j DtdbqbiDvdarah#,"tsfqoale#:"aicvdaq@n`jl-rv"}]
`const [newPassword, setNewPassword] = useState<[]>([]);_ 
This symbol '}' give me bug. How can I give params here to use this hash without this symbol '}' ?

**Comment by CodePapi** [comment]
Hi @ArmanBulatovich, Sorry about that.
I will try to reproduce this and get back to you

**Reply by arman-bolatuly** [reply]
Hi, Thank you! I will be very happy. This library is very good. Thanks for developing!!!

**Reply by CodePapi** [reply]
Hi @ArmanBulatovich ,
If it is ok, kindly email me @ oshiesam@gmail.com, so we can see to this bug and resolve together as I could not reproduce
